governance.md

```diff
@@ -293,42 +294,27 @@ building proprietary products and services that leverage Project Software and
 Services. Funding acquired by Institutional Partners to work on The Project is
 called Institutional Funding. However, no funding obtained by an Institutional
 Partner can override The Project BDFL and Steering Council. If a Partner has
-funding to do Jupyter/IPython work and the Council decides to not pursue that
+funding to do Matplotlib work and the Council decides to not pursue that
 work as a project, the Partner is free to pursue it on their own. However in
 this situation, that part of the Partner’s work will not be under the
-Jupyter/IPython umbrella and cannot use the Project trademarks in a way that
+Matplotlib umbrella and cannot use the Project trademarks in a way that
 suggests a formal relationship.
 
-To acknowledge institutional contributions, there are two level of Institutional
-Partners, with associated benefits:
+To acknowledge institutional contributions, Institutional Partners
+receive the following benefits:
 
-**Tier 1** = an institution with at least one Institutional Council Member
-
--   Acknowledged on the Jupyter/IPython websites, in talks and T-shirts.
--   Ability to acknowledge their own funding sources on the Jupyter/IPython
+-   Acknowledged on the Matplotlib websites, in talks and T-shirts.
+-   Ability to acknowledge their own funding sources on the Matplotlib
     websites, in talks and T-shirts.
--   Unlimited participation in the annual Institutional Partners Workshop, held
-    during the (planned) annual Jupyter Project Retreat. This allows the
-    Institutional Partner to invite as many of their own employees and funding
-    sources and collaborators as they want, even if they are not project
-    Contributors or Council Members.
 -   Ability to influence the project through the participation of their Council
     Member.
--   Council Members are invited to the bi-annual Jupyter/IPython Developer Meeting.
-
-**Tier 2** = an institution with at least one Institutional Contributor
-
--   Same benefits as Tier 1 level Partners, but:
--   Only Institutional Contributors are invited to the Institutional Partners
-    Workshop and bi-annual Jupyter/IPython Developer Meeting.
-
 
 Changing the Governance Documents
 =================================
 
 Changes to the governance documents are submitted via a GitHub pull
 request to The Project's governance documents GitHub repository at
-[https://github.com/jupyter/governance](https://github.com/jupyter/governance).
+[https://github.com/matplotlib/governance](https://github.com/matplotlib/governance).
 The pull request is then refined in response to public comment and
 review, with the goal being consensus in the community.  After this
```

Comment:
Probably should add a line to the effect of "For substantial changes, a notice will be posted to the matplotlib-users mailing list"?